assets/scss/global/_variables.scss

```diff
@@ -141,6 +141,9 @@ $heading:               #004d53;
 $step-number:           #a7a7a7;
 $brown-header:          #5c1f10;
 $single-bg:             #f6f4e7;
+$table-header:          #004466;
```

Comment:
I color picked the one shown in: https://jira.greenpeace.org/browse/PLANET-2702
`$dark-blue` is the closest, I would prefer to use that too, since I'd rather stop adding colors to this file, which has many already.

Reply:
Should we use `$dark-blue` then?

Reply:
I didn't try to colorpick, but that's my understanding from Will's comment. That we should use `#074365` for table header.

Reply:
Done!